packages/codegen/src/generateEntityCodegenFile.ts

```diff
@@ -227,7 +227,7 @@ export function generateEntityCodegenFile(config: Config, meta: EntityDbMetadata
       ${[o2m, m2o, o2o, m2m]}
 
       constructor(em: ${EntityManager}, opts: ${entityName}Opts) {
-        ${hasDefaultValues ? code`super(em, ${metadata}, {...${defaultValuesName}})` : code`super(em, ${metadata})`};
+        super(em, ${metadata}, ${hasDefaultValues ? `${defaultValuesName}` : "{}"}, opts);
```

Comment:
I was originally making some real changes here that I had to back out (thought I could call setOpts from the BaseEntity cstr, which I try like once every 3 months and forget that it can't b/c the subclass fields need to be instantiated first), but kept this minor change.